ORK1Kit/ORK1Kit/Consent/ORK1ConsentReviewStepViewController.m

```diff
@@ -219,9 +219,9 @@ - (ORK1ConsentReviewController *)makeDocumentReviewViewController {
     }
     
     NSString *html = [document mobileHTMLWithTitle:ORK1LocalizedString(@"CONSENT_REVIEW_TITLE", nil)
```

Comment:
Should the `CONSENT_REVIEW_TITLE` title text also be hidden when hideDefaultHeader is true? My (uncertain) understanding of "just lets the consent webview stretch to the top of the page" in https://github.com/CareEvolution/Consumers/issues/9957 is that the title should also be hidden.

Reply:
Yes...the "Review" title should be hidden...the content content should just stretch to the top

Reply:
Fixed. Updated screenshots.